.travis.yml

```diff
@@ -35,9 +42,7 @@ env:
 install:
   - if [ ! -d download-cache ]; then mkdir download-cache; fi
   - if [ ! -f download-cache/openssl-$OPENSSL_VER.tar.gz ]; then wget -O download-cache/openssl-$OPENSSL_VER.tar.gz https://www.openssl.org/source/openssl-$OPENSSL_VER.tar.gz; fi
-  - sudo apt-get install -qq -y cpanminus axel
   - sudo cpanm --notest Test::Nginx Protocol::WebSocket > build.log 2>&1 || (cat build.log && exit 1)
-  - wget http://nginx.org/download/nginx-${NGINX_VERSION}.tar.gz
```

Comment:
How do we download nginx in the first place? I think we should prepare for an empty cache, right?